README.rst

```diff
@@ -40,8 +40,9 @@ respecting the standard procedures in each system:
 * Use standard configuration space in ``/etc`` on traditional production servers
 * Handle environment-based configuration for cloud-based platforms
 
-``getconf`` v1.6 onwards supports 2.7, 3.3, 3.4, 3.5, 3.6 and is distributed under the two-clause BSD license.
```

Comment:
Small note: the way this sentence is worded gives the impression that there was a license change in version 1.10?

Reply:
Good point, I'll mention the license separately.

Reply:
... and while I'am at it, I'll add a `LICENSE` file for clarity.